tox.ini

```diff
@@ -13,8 +13,7 @@ deps = -r requirements/test.txt
 setenv =
   PYTHONPATH = {toxinidir}{/}install
   JUPYTER_PLATFORM_DIRS = 1
-  PYTHONWARNDEFAULTENCODING = 1
-commands = python -m pytest -n auto -v {posargs:tests}
+commands = python -X PYTHONWARNDEFAULTENCODING -m pytest -n auto -v {posargs:tests}
```

Comment:
I changed this because mypy raises encoding warnings when the env var is set. See also  #3154

Reply:
Why does mypy raise encoding warnings when the env var is set? :thinking: